ovs/flow_test.go

```diff
@@ -328,6 +328,8 @@ func TestFlowMarshalText(t *testing.T) {
 						&LearnedFlow{
 							DeleteLearned:  true,
 							FinHardTimeout: 1,
+							HardTimeout:    30,
```

Comment:
it'd be good to add a test for the backward compatible case where these values are not specified.

Reply:
woot, adding it